tests/saitests/py3/sai_qos_tests.py

```diff
@@ -341,6 +341,16 @@ def runTest(self):
                                            self.dst_port_3_ip, '192.168.0.1', '00:00:00:00:00:00', self.dst_vlan_3)
             send_packet(self, self.dst_port_3_id, arpreq_pkt)
 
+            for dut_i in self.test_port_ids:
+                for asic_i in self.test_port_ids[dut_i]:
+                    for dst_port_id in self.test_port_ids[dut_i][asic_i]:
+                        dst_port_ip = self.test_port_ips[dut_i][asic_i][dst_port_id]
+                        dst_port_mac = self.dataplane.get_mac(0, dst_port_id)
+                        arpreq_pkt = construct_arp_pkt('ff:ff:ff:ff:ff:ff', dst_port_mac,
+                                                       1, dst_port_ip['peer_addr'], '192.168.0.1',
+                                                       '00:00:00:00:00:00', None)
+                        send_packet(self, dst_port_id, arpreq_pkt)
+
```

Comment:
@XuChen-MSFT I recall you did some enhancement in the ARP learning for the QoS SAI test. Can you please help review this change? Thanks